net/local/local_bind.c

```diff
@@ -58,43 +58,43 @@ int psock_local_bind(FAR struct socket *psock,
               unaddr != NULL && unaddr->sun_family == AF_LOCAL &&
               addrlen >= sizeof(sa_family_t));
 
+  if (addrlen <= sizeof(sa_family_t))
```

Comment:
how this co-exists with `addrlen >= sizeof(sa_family_t)` in `DEBUGASSERT` above?

Reply:
let's remove the check

Reply:
Ok, but anyway, thy `<=` here and not `<`?

Reply:
since th caller must specify a non-empty path.

Reply:
But then shouldn't it be `if (addrlen <= sizeof(sa_family_t)) + 1`?
Or I misunderstand non-empty path?

Reply:
Done.